musicbot/bot.py

```diff
@@ -1396,6 +1396,35 @@ def _gen_embed(self):
 
         song_url = song_url.strip('<>')
 
+        fullPath = song_url+" " + ' '.join(leftover_args)
+        fullPath = fullPath.strip('"')
+        print(fullPath)
+        #print(os.path.exists(fullPath))
+        
+        if os.path.exists(fullPath): #check if given url is a path in file system.
+            print("found local song")
+            head, tail = os.path.split(fullPath)
+            print(head, tail)
+            self.downloader = downloader.Downloader(download_folder=head) #redudant?
+            player.playlist.downloader = self.downloader #redudant?
+            
+            if(os.path.isfile(fullPath)): #playing a file adds the file to the playlist.
+                entry, position = await player.playlist.add_entry_local(fullPath, channel=channel, author=author)
+            elif(os.path.isdir(fullPath)): #playing a folder adds whole content to playlist
+                for file in os.listdir(fullPath):
+                    if file.endswith((".mp3", ".wav", ".flac", ".ogg", ".wma")):
+                        entry, position = await player.playlist.add_entry_local(os.path.join(fullPath, file), channel=channel, author=author)
+            return Response("playing local", delete_after=30)
```

Comment:
Should respond very similarly to a normal song. i.e. using embeds would look like [this](https://cdn.mattbsg.xyz/bd93af32db0b1afc.png) instead of returning predefined text